src/fides/api/ops/service/privacy_request/request_runner_service.py

```diff
@@ -516,3 +548,75 @@ def generate_id_verification_code() -> str:
     Generate one-time identity verification code
     """
     return str(random.choice(range(100000, 999999)))
+
+
+def _retrieve_child_results(  # pylint: disable=R0911
+    fides_connector: Tuple[str, ConnectionConfig],
+    rule_key: str,
+    access_result: Dict[str, List[Row]],
+) -> Optional[List[Dict[str, Optional[List[Row]]]]]:
+    """Get child access request results to add to upload."""
+    try:
+        connector = FidesConnector(fides_connector[1])
+    except Exception as e:
+        logger.error(
+            "Error create client for child server %s: %s", fides_connector[0], e
+        )
+        return None
+
+    results = []
+
+    for key, rows in access_result.items():
+        if key.startswith(
+            f"{fides_connector[0]}:"
```

Comment:
just seeing if i can quickly address this

Reply:
should be addressed in https://github.com/ethyca/fides/pull/1894/commits/d0d6e76483b817d103fb392f100dfe8643215f8b